tutorials/deutsch_jozsa/deutsch_jozsa.ipynb

```diff
@@ -0,0 +1,393 @@
+{
```

Comment:
<p>Same note fore this image (to include in the PR or as a link to the AWS server - assure with Eden)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/179/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
I added the media on AWS under the deutsch-jozsa folder in classiq-docs-images bucket and used the <img> tag on the jupyter cell but not sure why you're unable to view it. Will check this!

Reply:
<p>Might be a ReviewNB issue if it's works locally</p>

Reply:
Ok that's good then because I'm able to view. Resolving this comment.